### PR TITLE
[RHPAM-2522] - Provide usable way of setting extensions image named namespace for external DB driver in Operator UI

### PR DESCRIPTION
**Pull request by swati-kale**
updated the Console form UI  w.r.t changes against RHPAM-2533 (build config changes in Operator for extensions image settings)
Signed-off-by: Swati Kale <swkale@redhat.com>

### CODE REVIEW

File: deploy/ui/form.json [hunk 1]
```diff
@@ -1122,6 +1122,50 @@
...
+                                  "displayWhen": "true",
+                                  "visible": false,
+                                  "fields": [
+                                    {
+                                      "label": "Extension image stream tag",
+                                      "type": "text",
+                                      "required": true,
+                                      "jsonPath": "$.spec.objects.servers[*].build.extensionImageStreamTag",
+                                      "description": "ImageStreamTag definition for the image containing the drivers and configuration. For example, custom-driver-image:7.7.0.",
+                                      "default": "rhpam-kieserver-library=org.openshift.quickstarts:rhpam-kieserver-library:1.5.0-SNAPSHOT"
```
Comment (spolti) [inline deploy/ui/form.json]: what does this default field means here?
it is the default value for hte kie container deployment id, it should not be here.

File: deploy/ui/form.json [hunk 2]
```diff
@@ -1122,6 +1122,50 @@
...
+                                    },
+                                    {
+                                      "label": "Extension image stream tag namespace",
+                                      "type": "text",
+                                      "required": false,
+                                      "jsonPath": "$.spec.objects.servers[*].build.extensionImageStreamTagNamespace",
+                                      "description": "Namespace within which the ImageStream definition for the image containing the drivers and configuration is located. Defaults to openshift namespace.",
+                                      "default": "rhpam-kieserver-library=org.openshift.quickstarts:rhpam-kieserver-library:1.5.0-SNAPSHOT"
```
Comment (spolti) [inline deploy/ui/form.json]: same than above.

File: deploy/ui/form.json [hunk 3]
```diff
@@ -1122,6 +1122,50 @@
...
+                                    },
+                                    {
+                                      "label": "Extension image install Dir",
+                                      "type": "text",
+                                      "required": false,
+                                      "jsonPath": "$.spec.objects.servers[*].build.extensionImageInstallDir",
+                                      "description": "scription: Full path to the directory within the extensions image where the extensions are located (e.g. install.sh, modules/, etc. Do not change this field unless you have sure about the changes that you is doing).",
+                                      "default": "rhpam-kieserver-library=org.openshift.quickstarts:rhpam-kieserver-library:1.5.0-SNAPSHOT"
```
Comment (spolti) [inline deploy/ui/form.json]: same than above.

File: deploy/ui/form.json (hunk 1 above)
Comment (swati-kale) [reply deploy/ui/form.json]: Yes, I will make the corrections

Comment (swati-kale) [reply deploy/ui/form.json]: @spolti changes are done.

File: deploy/ui/form.json (hunk 2 above)
Comment (swati-kale) [reply deploy/ui/form.json]: @spolti changes are done.

File: deploy/ui/form.json (hunk 3 above)
Comment (swati-kale) [reply deploy/ui/form.json]: @spolti changes are done.

File: deploy/ui/form.json [hunk 4]
```diff
@@ -1122,6 +1122,47 @@
...
+                                      "required": false,
+                                      "jsonPath": "$.spec.objects.servers[*].build.extensionImageStreamTagNamespace",
+                                      "description": "Namespace within which the ImageStream definition for the image containing the drivers and configuration is located. Defaults to openshift namespace."
+                                    },
+                                    {
+                                      "label": "Extension image install Dir",
+                                      "type": "text",
+                                      "required": false,
+                                      "jsonPath": "$.spec.objects.servers[*].build.extensionImageInstallDir",
+                                      "description": "scription: Full path to the directory within the extensions image where the extensions are located (e.g. install.sh, modules/, etc. Do not change this field unless you have sure about the changes that you is doing)."
```
Comment (ruromero) [inline deploy/ui/form.json]:
```suggestion
                                      "description": "Full path to the directory within the extensions image where the extensions are located (e.g. install.sh, modules/, etc.)."
```

File: deploy/ui/form.json [hunk 5]
```diff
@@ -1122,6 +1122,47 @@
...
+                                    },
+                                    {
+                                      "label": "Extension image stream tag namespace",
+                                      "type": "text",
+                                      "required": false,
+                                      "jsonPath": "$.spec.objects.servers[*].build.extensionImageStreamTagNamespace",
+                                      "description": "Namespace within which the ImageStream definition for the image containing the drivers and configuration is located. Defaults to openshift namespace."
+                                    },
+                                    {
+                                      "label": "Extension image install Dir",
```
Comment (ruromero) [inline deploy/ui/form.json]:
```suggestion
                                      "label": "Extension image install directory",
```

File: deploy/ui/form.json [hunk 6]
```diff
@@ -1122,6 +1122,47 @@
...
+                                  "fields": [
+                                    {
+                                      "label": "Extension image stream tag",
+                                      "type": "text",
+                                      "required": true,
+                                      "jsonPath": "$.spec.objects.servers[*].build.extensionImageStreamTag",
+                                      "description": "ImageStreamTag definition for the image containing the drivers and configuration. For example, custom-driver-image:7.7.0."
+                                    },
+                                    {
+                                      "label": "Extension image stream tag namespace",
```
Comment (ruromero) [inline deploy/ui/form.json]:
```suggestion
                                      "label": "Extension image stream namespace",
```

File: deploy/ui/form.json [hunk 7]
```diff
@@ -1122,6 +1122,47 @@
                           "displayWhen": "external",
                           "visible": false,
                           "fields": [
+                             {
```
Comment (ruromero) [inline deploy/ui/form.json]:
```suggestion
                            {
```

File: deploy/ui/form.json [hunk 8]
```diff
@@ -1122,6 +1122,47 @@
...
+                              "label": "Enable Extension image stream",
+                              "type": "checkbox",
+                              "required": false,
+                              "description": "When set to true will configure the KIE Server with Extension Image Stream.",
+                              "jsonPath": "$.spec.objects.servers[*].build",
+                              "originalJsonPath": "$.spec.objects.servers[*].build",
+                              "fields": [
+                                  {
```
Comment (ruromero) [inline deploy/ui/form.json]:
```suggestion
                                {
```

File: deploy/ui/form.json (hunk 4 above)
Comment (swati-kale) [reply deploy/ui/form.json]: I will do the changes, Do you want  the changes in CRD too (https://github.com/kiegroup/kie-cloud-operator/blob/master/deploy/crds/kieapp.crd.yaml#L253)

Comment (spolti) [reply deploy/ui/form.json]: please do.

Comment (swati-kale) [reply deploy/ui/form.json]: Done.

**Pull request merged**